templates/people.html

```diff
@@ -47,208 +48,67 @@
-                </div>
-                <div class="people-role-description">
-                    <div class="people-role people-role-top-level people-role-sme">SMEs</div>
-                    <div class="people-role-description-text">(Subject Matter Experts) have proven themselves to be experts in a given development area. They help guide Bevy's direction in their area and they are great people to reach out to if you have questions about a given area.</div>
-                </div>
-                <div class="people-role-description">
-                    <div class="people-role-description-text">Other members, without a special role, are part of the GitHub organization. They have the power to manage GitHub issues and pull requests.</div>
-                </div>
-            </div>
-        </p>
-        </div>
-        {% endif %}
-
-        {% if section.title == "The Bevy Organization" %}
-        <a class="anchor-target" id="org-members"></a>
-        <h2>
-            Members<a class="anchor-link" href="#org-members">#</a>
-        </h2>
-        {% endif %}
-
-        {% if section.title == "Community Members" %}
-        <div class="people-section-description media-content">
-        <p>
-            People in the Bevy community who are not part of the organization.
-        </p>
-        <p>
-            Anyone can add themselves here! <a href="https://github.com/bevyengine/bevy-community">Submit a pull request</a>!
-        </p>
-        </div>
-        {% endif %}
-
-        {% if section.pages %}
-        <div class="item-grid">
-            {% set pages = section.pages %}
-            {% if section.extra.sort_order_reversed %}
-            {% set pages = section.pages | reverse %}
-            {% endif %}
-            
-            {% for post in pages %}
-
-            <div class="card people-card" id="{{ post.title | slugify }}">
-                <div class="card-text media-content card-dense">
-                    <div class="card-title">
-                        <div style="width: 210px; font-size: {% if post.title | length > 30 %} 0.5rem; {% elif post.title | length > 20 %} 1.0rem; {% else %} 1.5rem; {% endif %}">
-                            {{ post.title }}
-                        </div>
-                        {% if post.extra.profile_picture %}
-                            <div class="card-profile-picture">
-                                {% if post.extra.profile_picture is starting_with("https://") %}
-                                    <img src="{{ post.extra.profile_picture }}" />
-                                {% else %}
-                                    <img src="{{ get_url(path=post.extra.profile_picture) }}" />
-                                {% endif %}
-                            </div>
-                        {% endif %}    
+            {% set members = load_data(path="org_members.json") %}
```

Comment:
It is required in practice though? This page will be empty if we don't generate the JSON files

Reply:
to build the full website yup, but for people who only want to work on part of it, like the book, they don't need it and we want to avoid the need for them to run the script